lib/svg-sprite/mode/symbol.js

```diff
@@ -137,9 +137,10 @@ SVGSpriteSymbol.prototype.layout = function(files, cb) {
                         removeAttributes.push(attribute.name);
```

Comment:
There's one `Array.prototype.forEach.call` here but I couldn't get rid of it last time I tried, and we don't have tests for this mode yet.